addons/l10n_be_edi/models/account_invoice.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api, _
+
+import base64
+
+from odoo.tools.xml_utils import check_with_xsd
+
+
+PATH_TO_XSD = 'l10n_be_edi/data/xsd/2.1/maindoc/UBL-Invoice-2.1.xsd'
+
+
+def to_object(item):
+    '''Helpers to convert recursively item to python object to easily access data
+    in the template to render.
+    item[k] = v => item.k = v
+    [dic1, dic2, ...] => [obj1, obj2, ...]
+    
+    :param item: The item to convert.
+    :return: a python object.
+    '''
+    def to_obj(item):
+        if isinstance(item, dict):
+            return type('temp', (), {k: to_obj(v) for k, v in item.iteritems()})
+        if isinstance(item, list):
+            return [to_obj(value) for value in item]
+        else:
+            return item
+
+    return to_obj(item)
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    @api.multi
+    def l10n_be_edi_create_values(self):
```

Comment:
Does this need to be callable over RPC? If not, it should be prefixed by an `_` and the decorator should be removed.

Also the name isn't great, it makes the reader think records will be created which is not the case.